pkg/platform/vcd/vcd.go

```diff
@@ -107,12 +115,6 @@ func (v *VcdPlatform) InitProvider(ctx context.Context, caches *platform.Caches,
 		}
 		log.SpanLog(ctx, log.DebugLevelInfra, "InitProvider", "mexInternalNetRange", mexInternalNetRange)
 
-		log.SpanLog(ctx, log.DebugLevelInfra, "InitProvider update isonet metadata", "stage", stage)
-		err = v.UpdateLegacyIsoNetMetaData(ctx)
```

Comment:
Is this really not used anymore? Since we don't have vcd setup to test, how can we be sure?

Reply:
There's this comment in vcd-network.go: `// on (re)start attempt ensure legacy (prior to common shared lb) ISO nets are in metadata`, so I'm pretty sure this isn't relevant anymore.